scss/_functions.scss

```diff
@@ -95,7 +95,7 @@
 // Color contrast
 // See https://github.com/twbs/bootstrap/pull/30168
 
-// A list of pre-calculated numbers of pow(($value / 255 + .055) / 1.055, 2.4). (from 0 to 255)
+// A list of pre-calculated numbers of pow(($value * .555 + .055) / 1.055, 2.4). (from 0 to 255)
 // stylelint-disable-next-line scss/dollar-variable-default, scss/dollar-variable-pattern
```

Comment:
This is wrong. / 255 is not the same as *.555. It should be * .00392

Reply:
> looks like `/ 2`1 got replaced by `* .5`1
I think the same thing happened here.